config/routes/fos_js_routing.yaml

```diff
@@ -1,6 +1,13 @@
 fos_js_routing:
     resource: "@FOSJsRoutingBundle/Resources/config/routing/routing.xml"
 
+# @deprecated
+edit_program_name:
```

Comment:
Do we need this? Can you try without these lines? 
My understanding is that I've already defined this route here
https://github.com/danielzhang130/Catroweb/blob/edd9ae597e90653f1cd2ba2baee89796300d5336/src/Catrobat/Controller/Web/ProgramController.php#L335

Reply:
I tried it without the lines and I got an error and it didn't go through. It looks like the other calls that use the route do the same as done here so I am guessing it is okay. It would be nice if we didn't have to specify this